src/main/resources/log4j2_indev.xml

```diff
@@ -2,16 +2,16 @@
 <Configuration status="WARN" packages="org.lanternpowered.server.console.launch">
     <Appenders>
         <TerminalConsole name="Console">
-            <PatternLayout pattern="[%d{HH:mm:ss}] [%t/%level] [%logger{1}]%fqcn{ [%path]}: %msg%n"/>
```

Comment:
@Meronat You removed the `FullLocationPatternConverter` formatting, you should readd the `%fqcn{ [%path]}`